pom.xml

```diff
@@ -152,9 +152,9 @@
       <version>1.4</version>
     </dependency>
     <dependency>
-    	<groupId>com.ibm.icu</groupId>
-    	<artifactId>icu4j</artifactId>
-    	<version>56.1</version>
+      <groupId>com.ibm.icu</groupId>
+      <artifactId>icu4j</artifactId>
+      <version>56.1</version>
```

Comment:
Unrelated. Removing tabs indentation.

Reply:
As a rule, it's a good thing to fix those things, but by filing a separate PR for such changes to avoid mixing up things and slowing down reviews IMO.

Reply:
Really a PR for that?